src/sections/DoYouNeedService/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "gatsby";
+
+import DoYouNeedWrapper from "./doYouNeedService.style";
+
+const DoYouNeedService = ({onClick}) => {
+    return (
+        <DoYouNeedWrapper>
+            <div id="content">
+                <div id="text">
+                    <span className="heading">Do You Need A <br/>Service Mesh?</span>
```

Comment:
Don't use <br> here. It affects the mobile view. Instead, define the width of this element to fit only the number of words you wish to see in one line.

Reply:
I agree on that too ✌️